lib/runtime/evaluation.coffee

```diff
@@ -16,19 +16,21 @@ module.exports =
     edpath = editor.getPath() || 'untitled-' + editor.getBuffer().id
     fn {editor, mod, edpath}
 
-  eval: ({move}={}) ->
+  eval: ({move, cell}={}) ->
```

Comment:
I kinda feel like cell evaluation is sufficiently different than selection/line eval, especially since the `move` behaviour needs to be different as well. Might make sense to create a new method, and maybe factor out the common parts?